AnnoyancesFilter/sections/popups.txt

```diff
@@ -112,6 +112,9 @@ bankbazaar.com#$#.oneTapLoginModal[style] { display: none !important; }
 bankbazaar.com#$#.modal-backdrop { display: none !important; }
 bankbazaar.com#$##js-sign-in-box[style="display: block;"] ~ .modal-backdrop { display: block !important; }
 magadanmedia.ru###social_modalblock
+inc42.com#$#.tp-modal { display: none !important; }
```

Comment:
It seems that it blocks a login popup.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/103275125-bcce0200-49c3-11eb-9274-0afbc094163e.png)

</details>